dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassFile.java

```diff
@@ -20,26 +20,25 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.MoreObjects;
-import java.nio.file.Path;
 import java.util.Objects;
 
 /**
- * A locator for a compiled class file of {@code className} in {@code jar} to uniquely locate the
- * class implementation in a class path.
+ * A locator for a compiled class file of {@code className} in {@code classPathEntry} to uniquely
```

Comment:
What's a "locator"? This is redundant with locate below. And perhaps this class is incorrectly named. I.e. why is it not `ClassFileLocator` if it represents a locator and not a class file.